.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-json.lock
+yarn.lock
```

Comment:
- [ ] I have to comments on this:

1. Even if you put `yarn.lock` here I can still see it in your repository since this was done after it was commited. In this case, you should have deleted it. Before thinking about doing it though, check my second comment below.
2. These files are supposed to be in your source control, meaning they should not be ignored. Please check this helpful comment about this: https://stackoverflow.com/a/44210813